modules/ocf_apphost/manifests/lets_encrypt.pp

```diff
@@ -13,7 +13,7 @@
       mode      => '0400';
   }
 
-  if $::hostname !~ /^dev-/ {
+  if !$::dev_config {
```

Comment:
did you confirm this is actually getting treated as a boolean? in the past, we always had to use `str2bool` for external facts like this, not sure if things have changed.